### DISCUSSION TITLE
Welcome to awesome-learning Discussions!

### DISCUSSION THREAD

**Discussion by mfaisalkhatri** [mfaisalkhatri/awesome-learning#42] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by IAnjaniKr** [comment]
I am concerned about the growing interference of AI in tech jobs in the enhancing and assistance, will it ultimately take up some types of tech jobs completely?

**Reply by mfaisalkhatri** [reply]
I don't think we should worry about the jobs, In fact we can learn more such tools like how to use them for testing and apply test automation tactics. I believe if we have the skills, we can get a job any time. Layoffs and recessions are temporary, once you learn a skill it is permanent.
